src/main/java/frc/utils/time/TimeUtils.java

```diff
@@ -32,8 +33,11 @@ public static void updateCycleTime() {
 	}
 
 	private static void logStatus() {
+		double currentTimeSeconds = getCurrentTimeSeconds();
+
 		Logger.recordOutput(TimeConstants.LOG_PATH + "CycleTimeSeconds", getCurrentCycleTimeSeconds());
-		Logger.recordOutput(TimeConstants.LOG_PATH + "CurrentTimeSeconds", getCurrentTimeSeconds());
+		Logger.recordOutput(TimeConstants.LOG_PATH + "CurrentTimeSeconds", currentTimeSeconds);
+		Logger.recordOutput(TimeConstants.LOG_PATH + "AverageCycleTimeSeconds", currentTimeSeconds / RobotManager.roborioCycles);
```

Comment:
make this function get roborioCycles as param instead if using it secretly like this